### DISCUSSION TITLE
Apple Cinema Display 27" working with 50% brightness or less with Dockstation Dell WD15

### DISCUSSION THREAD

**Discussion by wesleycacciari** [MonitorControl/MonitorControl#834] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Hey guys! 

I just got the dock station Dell WD15 to connect all devices (such as ACD 27", ethernet, USBs, charger) with one single Thunderbolt cable in my M1 MacBook Air and I'm facing the issue that the brightness doesn't go further than 50%. The DCC Max is set at 250 but it just don't work with the dockstation. 

The app works properly when I connect the monitor with MiniDP to USBc adapter.

Any help? 

Thanks in advance.

Wesley.

### Expected behavior

I expect the app will adjust the monitor brightness to max

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey 12.0.1
- Mac model: M1 MacBook Air
- MonitorControl version: 4.0.2 Built 6965
- Monitor(s): Apple Cinema Display
- Monitor Cable(s)/Connection(s): Monitor cable is Mini Displayport connect to Dockstation Dell WD15 and connected to Mac with Thunderbolt cable
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by waydabber** [comment]
Hi @wesleycacciari - by default the 0-50% brightness if the software component of the brightness scale if `Combine hardware and software dimming` is enabled. Probably DDC does not get through the W15 for some reason.

I'll convert this into a discussion so others with this dock might add their insights.
